trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/add-to-an-existing-payment-helper.ts

```diff
@@ -0,0 +1,28 @@
+import { SelectedFeeRecordsAvailablePaymentGroups } from '@ukef/dtfs2-common';
+
+export const getAvailablePaymentsHeading = (paymentGroups: SelectedFeeRecordsAvailablePaymentGroups) => {
+  if (!paymentGroups || paymentGroups.length === 0) {
+    throw new Error('No payment groups to select from.');
+  }
```

Comment:
We have this exact same check in the controller. Does the check for length there allow us to type this as a non-empty array to avoid the need to duplicate this check?